src/api/app/controllers/webui/user_controller.rb

```diff
@@ -76,8 +77,12 @@ def save
         redirect_back(fallback_location: root_path) && return
       end
     end
-    @displayed_user.realname = params[:realname]
-    @displayed_user.email = params[:email]
+
+    unless CONFIG[:ldap_mode]
```

Comment:
But I would expect that save does not work at all for ldap? What attributes would you change here?